pennylane/tape/__init__.py

```diff
@@ -17,5 +17,20 @@
 """
 from .tape import QuantumTape, get_active_tape, TapeError
 from .operation_recorder import OperationRecorder
-from .stop_recording import stop_recording
 from .unwrap import Unwrap, UnwrapTape
+
+
+def __getattr__(name):
```

Comment:
does the entire `__getattr__` need to be overwritten for this? Avoiding incorporating the `globals` indexing would be nice, plus many references to this will probably not be with `name == stop_recording`. If it's necessary it's probably not a big deal though

Reply:
Actually just rewrote this.

Reply:
Now `qml.tape.stop_recording` is a function context manager that wraps `qml.QueuingManager.stop_recording`.